velox/vector/tests/VectorMakerTest.cpp

```diff
@@ -458,6 +459,58 @@ TEST_F(VectorMakerTest, arrayVectorFromJson) {
   }
 }
 
+TEST_F(VectorMakerTest, nestedArrayVectorFromJson) {
+  auto arrayVector = maker_.nestedArrayVectorFromJson<int32_t>({
+      "[[1, 2], [2, 3, 4], [null, 7]]",
+      "[[1, 3, 7, 9], []]",
+      "[]",
```

Comment:
Would you add a case where empty array is followed by a non-empty array?

"[[1, 2, 3], [], [4, 5]]"